From 3eb24386719ee06c9e726550065c5427f1021c6f Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 13 Mar 2014 10:48:36 +0000
Subject: [PATCH 34/72] Update policy_tests.py for validate_plugin change

The policy_tests unit tests call validatePlugin(). Update them to
use the new name.

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 tests/policy_tests.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/policy_tests.py b/tests/policy_tests.py
index aa74da7..764af83 100644
--- a/tests/policy_tests.py
+++ b/tests/policy_tests.py
@@ -21,25 +21,25 @@ class PolicyTests(unittest.TestCase):
         p = FauxPolicy()
         p.valid_subclasses = []
 
-        self.assertTrue(p.validatePlugin(FauxPlugin))
+        self.assertTrue(p.validate_plugin(FauxPlugin))
 
     def test_redhat(self):
         p = FauxPolicy()
         p.valid_subclasses = [RedHatPlugin]
 
-        self.assertTrue(p.validatePlugin(FauxRedHatPlugin))
+        self.assertTrue(p.validate_plugin(FauxRedHatPlugin))
 
     def test_debian(self):
         p = FauxPolicy()
         p.valid_subclasses = [DebianPlugin]
 
-        self.assertTrue(p.validatePlugin(FauxDebianPlugin))
+        self.assertTrue(p.validate_plugin(FauxDebianPlugin))
 
     def test_fails(self):
         p = FauxPolicy()
         p.valid_subclasses = []
 
-        self.assertFalse(p.validatePlugin(FauxDebianPlugin))
+        self.assertFalse(p.validate_plugin(FauxDebianPlugin))
 
     def test_can_import(self):
         self.assertTrue(import_policy('redhat') is not None)
-- 
1.9.3

